cinch/group_vars/cent6

```diff
@@ -1,20 +1,20 @@
 gcc_compat_package: compat-gcc-34
 
 _repositories:
-  - name: epel-testing
-    mirrorlist: "{{ fedora_mirrors }}repo=testing-epel6"
-  - name: epel
-    mirrorlist: "{{ fedora_mirrors }}repo=epel-6"
-  - name: cloud
-    baseurl: "{{ centos_mirrors }}/$releasever/cloud/$basearch/openstack-juno/"
+- name: epel-testing
```

Comment:
I think we need to have a group discussion about using this indentation method for YAML. Personally, I am opposed to it, but I'm willing to have a discussion on it.